### PR TITLE
Add unified docs-ci

### PR DESCRIPTION
**Pull request by pedro-psb**
Related to: https://github.com/pulp/pulp-docs/issues/31

[noissue]

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -23,6 +23,7 @@ jobs:
         env:
           - TEST: pulp
           - TEST: docs
+          - TEST: new_docs
```
Comment (mdellweg) [inline .github/workflows/test.yml:26]: I do not believe this is in the plugin template. Please use the current version.

**Review by mdellweg — CHANGES_REQUESTED** [review]

File: .github/workflows/test.yml (hunk 1 above)
Comment (mdellweg) [reply .github/workflows/test.yml]: Not that this never happened to me...

**Review by mdellweg — APPROVED** [review]

**Comment by mdellweg** [comment]
I'm wondering if/when we want to stop building the docs the old way.

**Comment by pedro-psb** [comment]
We are planning to officially deactivate them on July, as a general strategy.
https://github.com/pulp/pulp-docs/issues/33#issuecomment-2050039066

**Review by ggainey — APPROVED** [review]

**Pull request merged**